modal_proto/api.proto

```diff
@@ -2586,6 +2586,7 @@ message SandboxWaitResponse {
 message Schedule {
   message Cron {
     string cron_string = 1;
+    string timezone = 2;
```

Comment:
We usually try to split out the proto changes into a separate PR first before adding the user facing SDK changes that use the new protos